posthog-core/src/index.ts

```diff
@@ -37,6 +37,7 @@ class PostHogFetchNetworkError extends Error {
 
   constructor(public error: unknown) {
     // TRICKY: "cause" is a newer property but is just ignored otherwise. Cast to any to ignore the type issue.
+    // eslint-disable-next-line @typescript-eslint/prefer-ts-expect-error
```

Comment:
otherwise `yarn lint` changes it every time.
if removing the ignore, the CI step `type-check` fails.